src/ServerTlsContext.php

```diff
@@ -7,9 +7,17 @@ final class ServerTlsContext
     public const TLSv1_0 = \STREAM_CRYPTO_METHOD_TLSv1_0_SERVER;
     public const TLSv1_1 = \STREAM_CRYPTO_METHOD_TLSv1_1_SERVER;
     public const TLSv1_2 = \STREAM_CRYPTO_METHOD_TLSv1_2_SERVER;
+    public const TLSv1_3 = \STREAM_CRYPTO_METHOD_TLSv1_3_SERVER ?? 64;
+
+    private const TLS_VERSIONS = [
+        'TLSv1.0' => self::TLSv1_0,
+        'TLSv1.1' => self::TLSv1_1,
+        'TLSv1.2' => self::TLSv1_2,
+        'TLSv1.3' => self::TLSv1_3,
+    ];
 
     /** @var int */
-    private $minVersion = \STREAM_CRYPTO_METHOD_TLSv1_0_SERVER;
+    private $minVersion = self::TLSv1_2;
```

Comment:
Why the BC break here?

Reply:
TLS 1.0 and 1.1 are EOL, so it would make sense to not default to supporting them.

Reply:
What are your thoughts here? This isn't an API break, but it is a behavior break in the name of better default security.

1. Set the default to TLS 1.2+ for servers and clients (I've been seeing this explicitly for some time now)
2. Allow TLS 1.0+ for servers, but require TLS 1.2+ for clients by default.
3. Revert this so servers and clients allow TLS 1.0+ and consider changing in the next major.

I don't have strong feelings on it – but AFAIK nothing should be using TLS 1.0/1.1 anymore.

I would like to get 1.3 support merged so my server can support it, so if this is a point of contention let's do number 3 and discuss changing the default minimum TLS version later.

Reply:
As we'll have a new major soon due to fibers, I'd defer this to the new major.